setups/provider.tf

```diff
@@ -0,0 +1,17 @@
+terraform {
+  required_version = ">= 0.7.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.26.0"
+    }
+  }
+}
+provider "aws" {
+  region = "eu-west-1"
+}
+
+provider "kubernetes" {
```

Comment:
please remove all provider blocks from module/sub-modules
all this blocks should go under project